activiti-spring-boot/spring-boot-starters/activiti-spring-boot-starter-hal-rest-api/src/main/java/org/activiti/services/ProcessInstanceController.java

```diff
@@ -92,4 +90,17 @@ public ProcessInstanceController(ProcessInstanceConverter processInstanceConvert
         runtimeService.signalEventReceived(signalInfo.getName(), signalInfo.getInputVariables());
         return new ResponseEntity<>(HttpStatus.OK);
     }
+
+    @RequestMapping(value = "{processInstanceId}/suspend")
+    public ResponseEntity<Void> suspend(@PathVariable String processInstanceId){
+        runtimeService.suspendProcessInstanceById(processInstanceId);
+        return new ResponseEntity<>(HttpStatus.OK);
+    }
+
+    @RequestMapping(value = "{processInstanceId}/activate")
```

Comment:
I'm wondering if `resume` is not a better name. Note: the internal status names are `suspended` and `active`.